.github/workflows/ci.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
       - main
       - master
+      - jacobq/*  
```

Comment:
I don't think this is needed -- it's just for where the target branch is

Reply:
like, what you're PR is going to be merged in to

Reply:
I just put that there so it would [run on my fork](https://github.com/jacobq/ember-apply/actions) (not needed any more since you authorized CI for this PR) without making a PR on the fork.

Reply:
ohhhh ok makes sense